src/components/bs5/tabs/tabs.data.json

```diff
@@ -0,0 +1,55 @@
+{
+    "type": "section",
```

Comment:
is it type ``section``, or type ``tabs``?

Expand to have the type prefix i.e.
``tab-container`` and ``tab-section``
or a new section like;
``type: tab``
``type-varient: section|container``

as you have added ``tab_variant`` which could be reduced to ``variant`` for standardization.